lib/datadog/lambda.rb

```diff
@@ -55,6 +55,11 @@ def self.wrap(event, context, &block)
         res = @listener.on_wrap(request_context: context, cold_start: cold) do
           block.call
         end
+      rescue NoMemoryError => e
+        # As of June 2020, AWS does not print NoMemoryError stack traces in Ruby
+        # So, we print the error string to logs so it can be handled by the logs forwarder
+        puts 'from Datadog Lambda Layer: failed to allocate memory (NoMemoryError)'
```

Comment:
hmm, this message could mislead the user to think it's the Datadog Lambda Layer caused the memory leak. What's the original message for `NoMemoryError`?